lib/sift/client.rb

```diff
@@ -34,7 +42,13 @@ def initialize(http_response, http_response_code)
     # == Returns:
     #   true on success; false otherwise
     def ok?
-      0 == @api_status.to_i
+      #if there is no content expected, use HTTP code
+      if Response::HTTP_CODES_WITH_NO_BODY.include? @http_status_code
+        204 == @http_status_code
+      # otherwise use API status
+      else
+        0 == @api_status.to_i 
```

Comment:
Good catch. Looks like the `to_i` part will make the method return `true` if the response body did not contain a status field.